test/test_cases/run_2d_moist_bubble.jl

```diff
@@ -114,11 +114,11 @@ function run_2d_moist_bubble(
         end
 
     elseif test_mode == :validation
-        temp_filepath = joinpath(@__DIR__, "moist_bubble_data")
+        temp_filepath = "/central/groups/esm/jiahe/debug-ClimaAtmos/moist_bubble_recip_removal_data/" 
```

Comment:
Can we avoid central/account-specific paths?

Reply:
Yes, this will be removed. I eventually got it run without crashing yesterday and just pushed without thinking. A lot of cleanup is needed